README.md

```diff
@@ -595,6 +595,32 @@ Untracked files:
 nothing added to commit but untracked files present (use "git add" to track)
 ```
 
+### masterAdm
```

Comment:
Would it be better to either a) Implement those other features using different commands, or b) Deal with that problem once it comes up?

Just concerned that right now all we need is a renumber so why not call it that. Later as new things are required we can look at introducing them and how to introduce them, but for now we don't have a use case.